.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Nit/optional: These comments are noise we probably don't need to keep in. We could leave in the link to the documentation if you want, but otherwise I'm not sure any of the comments here provide value.

Reply:
Done